README.md

```diff
@@ -172,7 +172,6 @@ Jump to section:
 |[SoundHound](https://soundhound.com/)|Voice interfaces|Santa Clara, CA|[SoundHound careers](soundhound.com/careers)|
 |[SpankChain](https://spankchain.com/)|Blockchain, adult industry, social justice|Los Angeles, CA|[SpankChain jobs](https://spankchain.com/jobs)|
 |[Square](https://squareup.com)|Financial services, mobile payments, payments hardware, and e-commerce|San Francisco, CA|[Square careers](https://squareup.com/ca/en/careers)|
-|[TELUS Digital](https://www.telus.com/)|Digital Labs for a major Canadian Telecom|Vancouver, BC|[TELUS Digital careers](https://labs.telus.com/careers)|
```

Comment:
I don't think it's a problem to put a specific department! For example we also have RBC Ventures, which is technically just a branch of RBC bank, and I think that's appropriate here.

However the careers link is broken and the main link just goes to the TELUS homepage. I think we should leave it in, but update the links to https://www.telus.com/en/digital (homepage for TELUS Digital) and https://www.telus.com/en/digital/careers (TELUS Digital careers)

Reply:
Hello:

I fixed the link for Telus's job link. At this PR https://github.com/toriagibbs/toronto-tech-scene/pull/90